pkg/controller/customdomain/customdomain_controller_test.go

```diff
@@ -257,115 +117,45 @@ func TestCustomDomainController(t *testing.T) {
-		Name:      "cluster",
-		Namespace: "",
-	}, actualIngressConfig)
+		Name:      instanceName,
+		Namespace: ingressOperatorNamespace,
+	}, actualCustomIngress)
 	if err != nil {
 		t.Fatalf("get ingress: (%v)", err)
 	}
-	if actualIngressConfig.Spec.Domain != domain {
-		t.Errorf(fmt.Sprintf("CRD ingresses.config.openshift.io/cluster domain mismatch: (%v)", actualIngressConfig.Spec.Domain))
+	if actualCustomIngress.Spec.Domain != userDomain {
+		t.Errorf(fmt.Sprintf("CRD ingresscontrollers.operator.openshift.io/default domain mismatch: (%v)", actualCustomIngress.Spec.Domain))
 	}
 
-	// check annotations
+	// check instance
 	actualCustomDomain := &customdomainv1alpha1.CustomDomain{}
 	err = r.client.Get(context.TODO(), types.NamespacedName{
-		Name:      name,
-		Namespace: namespace,
+		Name:      instanceName,
+		Namespace: instanceNamespace,
 	}, actualCustomDomain)
 	if err != nil {
 		t.Fatalf("get custom domain: (%v)", err)
 	}
-	if actualCustomDomain.ObjectMeta.Annotations == nil || actualCustomDomain.ObjectMeta.Annotations["original-domain"] != oldDomain {
-		t.Errorf(fmt.Sprintf("Problem with 'original-domain' annotation: (%+v)", actualCustomDomain.ObjectMeta.Annotations))
-	}
-	if actualCustomDomain.ObjectMeta.Annotations == nil || actualCustomDomain.ObjectMeta.Annotations["original-certificate"] != oldSecretName {
-		t.Errorf(fmt.Sprintf("Problem with 'original-certificate' annotation: (%+v)", actualCustomDomain.ObjectMeta.Annotations))
-	}
-
 	// check for ready status
 	if actualCustomDomain.Status.State != customdomainv1alpha1.CustomDomainStateReady {
 		t.Errorf(fmt.Sprintf("Status.State does not equal (%s)", string(customdomainv1alpha1.CustomDomainStateReady)))
 	}
 
-	// Check resulting modified 'system' routes
-	for n, ns := range systemRoutes {
-		actualRoute := &routev1.Route{}
-		expectedRoute := &routev1.Route{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      n,
-				Namespace: ns,
-			},
-			Spec: routev1.RouteSpec{
-				Host: n + "." + domain,
-			},
-		}
-		err = r.client.Get(context.TODO(), types.NamespacedName{
-			Name:      expectedRoute.Name,
-			Namespace: expectedRoute.Namespace,
-		}, actualRoute)
-		if err != nil {
-			t.Errorf("get customdomain: (%v)", err)
-			continue
-		}
-		if actualRoute.Name != expectedRoute.Name {
-			t.Error(fmt.Sprintf("actual route name (%v) does not match expected route name (%v)", actualRoute.Name, expectedRoute.Name))
-			continue
-		}
-		if actualRoute.Namespace != expectedRoute.Namespace {
-			t.Error(fmt.Sprintf("actual route namespace (%v) does not match expected route namespace (%v)", actualRoute.Namespace, expectedRoute.Namespace))
-			continue
-		}
-		if actualRoute.Spec.Host != expectedRoute.Spec.Host {
-			t.Error(fmt.Sprintf("actual route host (%v) does not match expected route host (%v)", actualRoute.Spec.Host, expectedRoute.Spec.Host))
-			continue
-		}
-	}
-
 	// Reconcile again so Reconcile() checks routes and updates the CustomDomain
```

Comment:
This second Reconcile is a no-op, right? Specifically, we're not checking routes, and we're not updating the CustomDomain?

Reply:
Yeah, we want to make sure that err is nil after the 2nd reconcile.

Reply:
Cool. TBC, I was pointing out that this comment is no longer accurate.

Reply:
Updated comments.